CAIPs/caip-312.md

```diff
@@ -0,0 +1,121 @@
+---
+caip: 312
+title: JSON-RPC Method for Retrieving Session Authorizations
+author: [Alex Donesky] (@adonesky1)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/285/files
+status: Draft
+type: Standard
+created: 2024-07-13
+requires: 25, 217
+---
+
+## Simple Summary
+
+CAIP-312 introduces the `wallet_getSession` method for retrieving authorizations from an active [CAIP-25][] initiated session.
+
+## Abstract
+
+This proposal aims to extend the [CAIP-25][] standard by defining a new JSON-RPC method for retrieving authorizations within a session. This method allows callers to dynamically retrieve authorizations and properties without necessarily having to persist and track it throughout the session's life.
```

Comment:
> This method allows callers to dynamically retrieve authorizations and properties without necessarily having to persist and track it throughout the session's life.

This is contradiction with updated language in CAIP-25 above.

Reply:
I think it would be improved if I just remove "without necessarily having to persist and track it throughout the session's life."

Reply:
done here: https://github.com/ChainAgnostic/CAIPs/pull/285/commits/ae0bcf4a33e720edaf39eb3c7e1dce08f9e68af0